examples/content_management_examples/public/plugin.ts

```diff
@@ -31,10 +31,17 @@ export class ContentManagementExamplesPlugin
       },
     });
 
+    contentManagement.registry.register({
+      id: 'todos',
+      version: {
+        latest: 'v1',
```

Comment:
WDYT about making the version a monotonically increasing integer? I.e:

```suggestion
        latest: 1,
```

Reply:
For some reason I liked better the `v1` format but I agree that changing it to a number makes sense. Will do that as part of my next PR